public/app/features/admin/LiveAdmin.tsx

```diff
@@ -99,19 +178,36 @@ export class LiveAdmin extends PureComponent<Props, State> {
             <br />
           </Container>
 
-          <h2>Channels</h2>
-          <Select options={channels} value={current} onChange={this.onChannelChanged} allowCustomValue={true} />
-          <br />
-
-          <LivePanel channel={channel} />
-
+          <div
```

Comment:
NOTE:  this page will be thrown out before it is actually released, so it is really just for debugging